amethyst_rendy/src/plugins.rs

```diff
@@ -3,14 +3,11 @@
 use crate::{
     bundle::{RenderOrder, RenderPlan, RenderPlugin, Target},
     pass::*,
-    sprite_visibility::build_sprite_visibility_sorting_system,
-    visibility::build_visibility_sorting_system,
+    sprite_visibility::{build_sprite_visibility_sorting_system, SpriteVisibility},
```

Comment:
These two new imports needed? Doesn't look like usage for them was added

Reply:
Yes, because now these resources are inserted by the rendering system bundles instead of the system itself.